<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdeWindow</class>
 <widget class="QMainWindow" name="MdeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="activationOrder">
       <enum>QMdiArea::StackingOrder</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuHistory">
     <property name="toolTip">
      <string>Recent closed files</string>
     </property>
     <property name="statusTip">
      <string>Recent closed files</string>
     </property>
     <property name="title">
      <string>History</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionOpenAllTheHistoryFiles"/>
     <addaction name="actionClearRecentHistory"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
    <addaction name="menuHistory"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuTab">
     <property name="title">
      <string>Tab</string>
     </property>
     <addaction name="actionNext"/>
     <addaction name="actionPrevious"/>
    </widget>
    <addaction name="menuTab"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionCmdlParam"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/new.png</normaloff>:/myImage/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new document</string>
   </property>
   <property name="statusTip">
    <string>Create a new document</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/open.png</normaloff>:/myImage/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing file</string>
   </property>
   <property name="statusTip">
    <string>Open an existing file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/save.png</normaloff>:/myImage/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current document</string>
   </property>
   <property name="statusTip">
    <string>Save current document</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/saveAs.png</normaloff>:/myImage/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save current document as a new file</string>
   </property>
   <property name="statusTip">
    <string>Save current document as a new file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/exit.png</normaloff>:/myImage/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClearRecentHistory">
   <property name="text">
    <string>Clear recent history</string>
   </property>
   <property name="toolTip">
    <string>Clear recent history</string>
   </property>
   <property name="statusTip">
    <string>Clear recent history</string>
   </property>
  </action>
  <action name="actionOpenAllTheHistoryFiles">
   <property name="text">
    <string>Open all the history files</string>
   </property>
   <property name="statusTip">
    <string>Open all the history files</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close the current tab</string>
   </property>
   <property name="statusTip">
    <string>Close the current tab</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="toolTip">
    <string>Close all tabs</string>
   </property>
   <property name="statusTip">
    <string>Close all tabs</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/about.png</normaloff>:/myImage/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About this application</string>
   </property>
   <property name="statusTip">
    <string>About this application</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/qt.png</normaloff>:/myImage/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/next.png</normaloff>:/myImage/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next tab</string>
   </property>
   <property name="statusTip">
    <string>Next tab</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/myImage/images/previous.png</normaloff>:/myImage/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous tab</string>
   </property>
   <property name="statusTip">
    <string>Previous tab</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="statusTip">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCmdlParam">
   <property name="text">
    <string>CommandLine Parameters</string>
   </property>
   <property name="toolTip">
    <string>Command Line Parameters</string>
   </property>
   <property name="statusTip">
    <string>Command Line Parameters</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myimage.qrc"/>
 </resources>
 <connections/>
</ui>
